src/components/Navbar/Navbar copy.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import "./Navbar.css"
+// import "./Nav.scss"
+import { Link } from "react-router-dom"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faShoppingCart } from "@fortawesome/free-solid-svg-icons"
+export default function Navbar() {
```

Comment:
why copy, do not push unnecessary things

Reply:
cleared it